examples/configuration/UsingSSHToConfigureNameResolution.ipynb

```diff
@@ -80,9 +131,9 @@
     "\n",
     "PUBLIC_NETWORK_ID=$(openstack network show public -c id -f value)\n",
     "\n",
-    "echo Creating network ${EXOGENI_STITCH_NAME}\n",
+    "echo Creating network ${NETWORK_NAME}\n",
     "blazar lease-create \\\n",
-    "   --physical-reservation min=${CLIENT_COUNT},max=${CLIENT_COUNT},resource_properties='[\"=\", \"$node_type\", \"compute_haswell\"]' \\\n",
+    "   --physical-reservation min=$((CLIENT_COUNT + 1 )),max=$((CLIENT_COUNT +1)),resource_properties='[\"=\", \"$node_type\", \"compute_haswell\"]' \\\n",
```

Comment:
This is me being psychotic, but can you please add a space between the last `+1`?